app/models/service.rb

```diff
@@ -362,8 +362,8 @@ def chargeback_report_name
 
   def generate_chargeback_report(options = {})
     _log.info "Generation of chargeback report for service #{name} started..."
-    MiqReportResult.where(:name => chargeback_report_name).destroy_all
-    report = MiqReport.new(chargeback_yaml)
+    MiqReport.where(:name => chargeback_report_name).destroy_all
+    report = MiqReport.create(chargeback_yaml)
```

Comment:
What happens if two chargeback reports run at the same time?  I don't understand the destruction of the history.  cc @gtanzillo